### PR TITLE
feat(components): Add layout components

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/v2/Layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
```
Comment (coreyar) [inline src/components/v2/Layout/Layout.tsx:1]: I think it is logical to call this file the index so we don't duplicate the directory name for the base file.

Comment (coreyar) [reply src/components/v2/Layout/Layout.tsx:1]: We should add a storybook of this component as well.

Comment (baomastr) [reply src/components/v2/Layout/Layout.tsx:1]: added index.ts for import without folder name

File: src/components/v2/Layout/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Box from '@mui/material/Box';
+import { Sidebar } from './Sidebar';
+import { Header } from './Header';
+import { PageContainer } from './PageContainer';
+
+/* sidebar width */
+const drawerWidth = 180;
```
Comment (coreyar) [inline src/components/v2/Layout/Layout.tsx]: Should this be defined as a theme constant and imported here?

File: src/components/v2/Layout/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    text: 'Dev',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+];
+
+interface IProps {
+  drawerWidth: number;
+}
+
+export const Sidebar = ({ drawerWidth }: IProps) => {
```
Comment (coreyar) [inline src/components/v2/Layout/Sidebar.tsx]: If the drawerWidth is part of the theme we don't need to pass it as a prop

File: src/components/v2/Layout/Layout.tsx (hunk 2 above)
Comment (baomastr) [reply src/components/v2/Layout/Layout.tsx]: this variable is used just in scope of layout, but if you think that we should define it inside theme, i would move it

File: src/components/v2/Layout/Sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  },
+  {
+    href: '/market',
+    text: 'Market',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/vault',
+    text: 'Vault',
+    icon: <XVSIcon width={20} height={20} />,
```
Comment (coreyar) [inline src/components/v2/Layout/Sidebar.tsx]: We probably want to have these values be a constant in the theme so we have consistent icons across the app

File: src/components/v2/Layout/Sidebar.tsx (hunk 3 above)
Comment (baomastr) [reply src/components/v2/Layout/Sidebar.tsx]: okay. gonna define it inside theme scope

File: src/containers/App.tsx [hunk 5]
```diff
@@ -75,6 +77,7 @@ class App extends React.Component {
                         {process.env.REACT_APP_CHAIN_ID === '97' && (
                           <Route exact path="/faucet" component={Faucet} />
                         )}
+                        {isDevEnvironment && <Route exact path="/dev" component={Dev} />}
```
Comment (coreyar) [inline src/containers/App.tsx]: What is this page for?

Comment (baomastr) [reply src/containers/App.tsx]: just for develop debugging

File: src/components/v2/Layout/Sidebar.tsx (hunk 4 above)
Comment (baomastr) [reply src/components/v2/Layout/Sidebar.tsx]: seems like this values may be constants just in scope of sidebar.
in other parts of app may be other sizes for icons

**Comment by baomastr** [comment]
stories for layout components are ready @coreyar

File: src/containers/App.tsx (hunk 5 above)
Comment (coreyar) [reply src/containers/App.tsx]: Let's remove this in favor of using storybooks. Storybooks are perfect for this purpose

Comment (coreyar) [reply src/containers/App.tsx]: And as a side effect we also use them for snapshotting so if we develop with them then there is no extra work to setup the snapshot

**Comment by coreyar** [comment]
> stories for layout components are ready @coreyar

It looks like the responsiveness of the [Dashboard ](https://tf-venus-prod-vizreg.s3.amazonaws.com/fe58ede84b0eedb881695d3310d3eeef0212e970/index.html?id=changed-chrome_iphone7_Dashboard_Dashboard.png)changed. The main thing I'd call out is the capitalization changed on the connect button.

**Comment by baomastr** [comment]
> > stories for layout components are ready @coreyar
> 
> It looks like the responsiveness of the [Dashboard ](https://tf-venus-prod-vizreg.s3.amazonaws.com/fe58ede84b0eedb881695d3310d3eeef0212e970/index.html?id=changed-chrome_iphone7_Dashboard_Dashboard.png)changed. The main thing I'd call out is the capitalization changed on the connect button.

yes, we don't need capitalization in buttons, like on current production version

**Comment by baomastr** [comment]
I also set up the `layout: 'fullscreen'` in preview.js in case of paddings in body for storybooks preview.

File: src/containers/App.tsx (hunk 5 above)
Comment (baomastr) [reply src/containers/App.tsx]: removed

**Review by coreyar — APPROVED** [review]

**Pull request merged**